responsible-ai-collaborative/aiid
# Add configurable right sidebar

Right Sidebar content can be passed to the `Layout` as a React fragment, resolving #951. Moves Similar Incidents to sidebar at appropriate screen sizes.

![image](https://user-images.githubusercontent.com/25443411/200654473-9112d0bc-5ab6-41d0-921e-f4846ba3b1c4.png)


## Breakage Attempts / Testing

- Check outline sidebar on multiple blog posts and mdx pages
- Check outline sidebar in Spanish
- Check similar incidents sidebar in multiple languages
- Check similar incidents sidebar in multiple browsers
  - **Found issue**: When the heading wraps, there's not enough line-height. Fixed. 
- Resize citation page 1 pixel at a time
  - **Found issue**: The bottom similar incidents don't have enough space at ~900px. Fixed.
- Check that at exactly 1279–1280 px, there is no point at which the sidebar and bottom similar incidents are both visible.
- Inspect the citation page as a whole
  - **Found issue**: the margins above the bottom similar incidents and below the subtitle look weird. Fixed.
  - **Found issue**: the heading above the reports looks weird with new titleWrapper styling. Fixed.
